ledfx/nowplaying/providers/mpris.py

```
@@ -0,0 +1,443 @@
+        """Connect to session D-Bus and set up player discovery hooks."""
+        try:
+            from dbus_fast.aio import MessageBus
+        except ImportError:
+            _LOGGER.debug("MPRIS: dbus-fast not installed; provider disabled")
+            return
+
+        try:
+            self._bus = await MessageBus().connect()
+        except asyncio.CancelledError:
+            raise
+        except Exception as exc:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)